static/readme.js

```diff
@@ -40,7 +40,13 @@ else {
     })
     .catch((err) => {
       console.error('Failed to load readme:', err)
-      target.innerHTML = '😒 the readme failed to load.'
+      if (source.includes('codeberg.org/') || source.includes('gitlab.com/')) {
+        target.style.display = 'none'
+      }
+      else {
+        target.style.textAlign = 'center'
+        target.innerHTML = '😒<br>the readme failed to load.'
```

Comment:
```suggestion
        target.innerHTML = '😒<br>The readme failed to load.'
```

<img width="291" height="114" alt="Image" src="https://github.com/user-attachments/assets/0ceb882e-1c5c-4e6b-94b0-e61bda94e0cf" />